connectors/api_entreprises.py

```diff
@@ -8,25 +7,25 @@ class ApiEntrepriseException(Exception):
     pass
 
 
-def get_by_offerer(offerer: Offerer) -> Response:
-    return get_by_siren(offerer.siren)
+def get_by_offerer(offerer: Offerer) -> dict:
+    return _get_by_siren(offerer.siren)
 
 
-def get_by_siren(siren: str) -> Response:
-    response = requests.get("https://entreprise.data.gouv.fr/api/sirene/v1/siren/" + siren,
+def _get_by_siren(siren: str) -> dict:
+    response = requests.get("https://entreprise.data.gouv.fr/api/sirene/v3/unites_legales/" + siren,
```

Comment:
```suggestion
    response = requests.get(f'https://entreprise.data.gouv.fr/api/sirene/v3/unites_legales/{siren}',
```
La fonction `f` est la dernière sortie pour faire une concaténation, c'est une bonne pratique de l'utiliser.